Neos.Neos/Documentation/CreatingASite/NodeTypes/DisableNodeTypes.rst

```diff
@@ -0,0 +1,35 @@
+.. _disable-nodetypes:
+
+Disable NodeTypes
+===================
+
+To hide an existing NodeType (i.e. one that comes with Neos already) you have 2 options.
+
+Hide the NodeType from the user interface
+=========================================
+
+*NodeTypes.yaml*
+
+.. code-block:: yaml
+
+  Vendor.Site:YourContentElementName:
+    ui: ~
+
+Nodes of this type will still remain valid in the database and being rendered to the frontend. But they will not be
+shown anymore in the dialog for adding nodes.
+
+Completely disallow the direct usage of a NodeType
+==================================================
+
+*NodeTypes.yaml*
+
+.. code-block:: yaml
+
+  Vendor.Site:YourContentElementName:
+    abstract: TRUE
+
+As abstract NodeTypes are not valid to be used directly this will hide the NodeType in the user interface AND
+additionally make all existing nodes of this type invalid. If you run a node:repair all existing nodes of this type will
```

Comment:
"node:repair" could be quoted with backticks to hightlight it as a command:
\`node:repair\`